Xamarin.Forms.Platform.WP8/VisualElementTracker.cs

```diff
@@ -272,9 +272,9 @@ void HandleRedrawNeeded(object sender, EventArgs e)
 
 		void MaybeInvalidate()
 		{
-			if (Model.IsInNativeLayout)
-				return;
-			var parent = (FrameworkElement)Element.Parent;
+           if (Model.IsInNativeLayout)
```

Comment:
Yea I had a similar issue on my other PR.  I cant seem to figure out why the tabs are not working correctly.  I have pulled down a fresh copy on two computers and the tabs are correct on my side when I pull.  Any ideas?